app/helpers/bill_parser.rb

```diff
@@ -0,0 +1,40 @@
+	module BillConverter
+
+		def parse_bill
+			bill_source = APP_ROOT.to_path+'/public/uploads/'+params[:bill][:name]
```

Comment:
Appending a user given string to a path is a security risk. Someone could post a bill name of '../../../../etc/passwords' and possibly get their hands on sensitive information.